packages/cozy-client/src/Query.jsx

```
@@ -1,9 +1,8 @@
-import { Component } from 'react'
+import React, { useEffect, useReducer, useRef } from 'react'
 import CozyClient from './CozyClient'
```

ESLint

[error] 'CozyClient' is defined but never used.

(no-unused-vars)